owners/src/github.js

```diff
@@ -240,14 +240,21 @@ class GitHub {
     );
     this.logger.debug('[getReviews]', number, response.data);
 
-    return response.data.map(
-      json =>
-        new Review(
-          json.user.login.toLowerCase(),
-          json.state,
-          new Date(json.submitted_at)
-        )
-    );
+    // See https://developer.github.com/v4/enum/pullrequestreviewstate/ for
+    // possible review states. The only ones we care about are "APPROVED" and
+    // "CHANGES_REQUESTED", since the rest do not indicate a definite approval
+    // or rejection.
+    const allowedStates = ['approved', 'changes_requested'];
+    return response.data
+      .filter(({state}) => allowedStates.includes(state.toLowerCase()))
```

Comment:
just a qq, do states actually change in case or are we just being defensive here cause well, github is never consistent LOL

Reply:
The are ''always'' capitalized lol. Being defensive here.